sound/pulseaudio/Makefile

```diff
@@ -30,7 +30,7 @@ define Package/pulseaudio-daemon/Default
   SECTION:=sound
   CATEGORY:=Sound
   DEPENDS:=+libsndfile +libltdl +libpthread +librt +alsa-lib \
-	+libopenssl +libcap $(ICONV_DEPENDS) $(INTL_DEPENDS)
+	+libopenssl +libcap +glib2 $(ICONV_DEPENDS) $(INTL_DEPENDS)
```

Comment:
$(ICONV_DEPENDS) $(INTL_DEPENDS) come with glib2. no need for them now.

Reply:
+libpthread +librt are also not needed too.

Reply:
libltdl might not be needed.

Reply:
Glib2 (GIO) was needed for it to build.

Reply:
That should be specified in the commit description!

Reply:
glib2 was patched out.